### ISSUE TITLE
Buttons white despite "dark" class being present

### ISSUE THREAD

**Issue by jakajancar** [Yonom/assistant-ui#959]
I'm setting `class="dark"` on the parent of `<Thread/>`. What am I doing wrong?

<img width="495" alt="Screenshot 2024-10-09 at 17 20 44" src="https://github.com/user-attachments/assets/aad28c34-170c-4663-b6ac-bf488bb0d323">

**Comment by Yonom** [comment]
Hey @jakajancar 

let's investigate why... Can you verify that your tailwind.config.js file contains the following?

```ts
const config = {
  darkMode: ["class"],
```

If that is not solving the issue, can you paste the output of the following command:
```ts
npx next info
```

A few more questions:
- are you using shadcn-ui?
- do you have any other Tailwindcss plguins installed?

**Comment by jakajancar** [comment]
Amm, I'm not using tailwind/shadcn-ui, I just did the imports as suggested in the Getting Started for "Not using Tailwind":

```
import "@assistant-ui/react/styles/index.css";
import "@assistant-ui/react/styles/modal.css";
```

I'm also not using `Next`, only pure React and `ai`.

**Comment by Yonom** [comment]
I found and fixed an issue regarding the borders when using the non-tailwind non-shadcn-ui themes.

But I am still struggling to find the cause of the button backgrounds...  I create a new nextjs project without tailwind/shadcn and installed the assistant-ui the way you described, and the buttons look normal to me.

What browser do you use? 

Do you have any other CSS styles that might be applying to buttons? 

Can you try to investigate where the current background color of the action bar buttons is coming from? (e.g. inspect element and search for "background-color" among the styles and make a screenshot of the class)

In my case, it is set to `transparent` from the following selector:

```
:where(.aui-root) button, 
:where(.aui-root) input:where([type=button]), 
:where(.aui-root) input:where([type=reset]), 
:where(.aui-root) input:where([type=submit])
```

<img width="1161" alt="Screenshot 2024-10-12 at 19 44 10" src="https://github.com/user-attachments/assets/6d6f9808-fef7-44ef-95c6-c33f7e9a9153">

**Comment by jakajancar** [comment]
Oh god 🤦🏻‍♂️ [Some leftover CSS](https://github.com/vitejs/vite/blob/4ab37fd6e5b0cd527b04b62ab18630b5e596c3e4/packages/create-vite/template-react/src/index.css#L65-L67) from the Vite template that I did not notice.

Apologies, I just assumed it was assistant-ui since it's an "empty" project, but not so empty after all.

**Comment by Yonom** [comment]
@jakajancar glad you found the root cause! 🚀 

Your issue helped spot a different bug with border colors not being applied correctly, fixed in @assistant-ui/react@0.5.78
Thank you for that 👍
